source/eventcore/driver.d

```diff
@@ -504,7 +504,7 @@ interface EventDriverFiles {
 		needs to make sure that either `on_read_finish` got called, or
 		`cancelRead` was called before issuing the next call to `read`.
 	*/
-	void write(FileFD file, ulong offset, const(ubyte)[] buffer, IOMode mode, FileIOCallback on_write_finish);
+	void write(FileFD file, ulong offset, return const(ubyte)[] buffer, IOMode mode, return FileIOCallback on_write_finish);
```

Comment:
Shouldn't this be `scope` instead? What does `return` mean when there is no return value? I guess I should do more research of what DIP1000 has become by now, but somehow I can't really force myself to do that.

Reply:
I think the `return` means that the parameter is then stored on the `this` object, since we are inside a class, although I haven't verified that it does make sense for that is how it works in the constructor and what it means to annotate the function itself as `return`

Reply:
after quick testing, it seems that `return` and `scope` on classes is a mess

but at least compilation works without warnings for now lol